src/content/reference/cli.mdx

```diff
@@ -557,6 +559,26 @@ You need to be [logged in](reference/cli.mdx#context) to Okteto before running t
 | _--volumes_   |  (bool)  | Destroy the persistent volumes created by the pipeline (defaults to false)                                                               |
 | _--wait_      |  (bool)  | Wait until the pipeline is destroyed (defaults to false)                                                                                 |
 
+#### list
+
+List all your okteto development environments in the namespace in use.
+
+```console
+$ okteto pipeline list
+```
+
+Run `okteto pipeline list` to get the status and info of your development environments.
+
+You need to be [logged in](reference/cli.mdx#context) to Okteto before running this command.
```

Comment:
```suggestion
You must be [logged in](reference/cli.mdx#context) to Okteto before running this command.
```

Reply:
I took the opportunity and I also updated other lines with the previously proposed text 019233be7630dbadfa770df3ee4cac41048bc42a